Justfile

```diff
@@ -105,7 +105,7 @@ docstrings: devenv
     $BIN/pydocstyle databuilder/backends/tpp.py
 
     # only enforce classes are documented for the public facing docs
-    $BIN/pydocstyle --add-ignore=D102,D103,D105,D106 databuilder/contracts/contracts.py
+    $BIN/pydocstyle --add-ignore=D102,D103,D105,D106 databuilder/contracts/wip.py
```

Comment:
Yes, me too. Particularly since we've now got, effectively, two different TPP schemas: the WIP one here, and the one I developed for the booster study. (They were originally different sorts of thing, but this PR makes them direct competitors.)

I wondered if it would be helpful to put things under a `databuilder.tables.beta` namespace while we're still working on them so it's clear to both us and users.

Reply:
> I wondered if it would be helpful to put things under a databuilder.tables.beta namespace

Yes, maybe it would.

I can't find the issue called something like "support future backwards incompatible changes" which envisaged some kind of versioning scheme. Do you know where that's gone, @inglesp?

Irrespective of namespacing, though, I think we should be brave in unifying what we've got asap, even if it breaks something.